### ISSUE TITLE
Leads to getaddrinfo to fail

### ISSUE THREAD

**Issue by yarikoptic** [nsntrace/nsntrace#28]
```
$> sudo nsntrace -d wlp82s0 --user $USER httpfs2 -f http://datasets.datalad.org/labs/churchland/najafi-2018-nwb/data/FN_dataSharing/nwb/mouse1_fni16_150819_001_ch2-PnevPanResults-170815-163235.nwb /mnt/httpfs
Starting network trace of 'httpfs2' on interface wlp82s0.
Your IP address in this trace is 172.16.42.255.
Use ctrl-c to end at any time.

file name: 	mouse1_fni16_150819_001_ch2-PnevPanResults-170815-163235.nwb
host name: 	datasets.datalad.org
port number: 	80
protocol: 	http
request path: 	/labs/churchland/najafi-2018-nwb/data/FN_dataSharing/nwb/mouse1_fni16_150819_001_ch2-PnevPanResults-170815-163235.nwb
auth data: 	(null)
httpfs2: getaddrinfo datasets.datalad.org - Name or service not known
Connection failed.
Finished capturing 44 packets.
```
Command works fine if I run without nsntrace.  The error when running without `--user`

<details>
<summary>some details of network interfaces</summary> 

```shell
$> ifconfig         
docker0: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
        inet 172.17.0.1  netmask 255.255.0.0  broadcast 172.17.255.255
        inet6 fe80::42:1aff:fe8b:ce25  prefixlen 64  scopeid 0x20<link>
        ether 02:42:1a:8b:ce:25  txqueuelen 0  (Ethernet)
        RX packets 1128  bytes 68306 (66.7 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 1233  bytes 16254976 (15.5 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
        inet 127.0.0.1  netmask 255.0.0.0
        inet6 ::1  prefixlen 128  scopeid 0x10<host>
        loop  txqueuelen 1000  (Local Loopback)
        RX packets 1374  bytes 145239 (141.8 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 1374  bytes 145239 (141.8 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

wlp82s0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.1.34  netmask 255.255.255.0  broadcast 192.168.1.255
        inet6 fe80::5435:e7b4:e4eb:2abd  prefixlen 64  scopeid 0x20<link>
        ether 94:e6:f7:72:7c:70  txqueuelen 1000  (Ethernet)
        RX packets 1802284  bytes 2072415424 (1.9 GiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 664687  bytes 153508175 (146.3 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


$> cat /etc/resolv.conf 
# Generated by NetworkManager
nameserver 192.168.1.1

$> route
Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
default         ESCAPE          0.0.0.0         UG    600    0        0 wlp82s0
link-local      0.0.0.0         255.255.0.0     U     1000   0        0 wlp82s0
172.17.0.0      0.0.0.0         255.255.0.0     U     0      0        0 docker0
192.168.1.0     0.0.0.0         255.255.255.0   U     600    0        0 wlp82s0

```
</details>
Version is 0~20160806-1+b1, but I have not spotted any possibly relevant changes in the git history here.

**Comment by jonasdn** [comment]
Hi!

What is the contents of your `/etc/resolv.conf`? Applications such as systemd-resolv or dnsmasq will sometime run nameserver on loopback addresses (127.x.x.x) and that will not work from the network namespace nsntrace creates.

**Comment by jonasdn** [comment]
Possibly closed by #33 please recheck with release 4 and the `--use-public-dns` option.
